apps/frontend/src/pages/contributors/libs/components/contributor-update-form/contributor-update-form.tsx

```diff
@@ -68,6 +73,21 @@ const ContributorUpdateForm = ({
 				label="Project"
 				name="projects"
 			/>
+			<div className={styles["checkbox-wrapper"]}>
+				<label className={styles["checkbox-label"]} htmlFor="isHidden">
+					<Checkbox
+						control={control}
+						errors={errors}
+						id="isHidden"
+						name="isHidden"
+					/>
+					<span>Is hidden</span>
```

Comment:
I understand that you didn't choose this name, but am I the only one who thinks "Is Hidden" sounds more like developer language than something suited for a user interface?
"Do Not Track", or perhaps just "Hidden"

Reply:
good suggestion, I like do not track option (on the backend side let's keep it as it is now)